crates/circuit/src/packed_instruction.rs

```diff
@@ -493,15 +493,15 @@ impl Drop for PackedOperation {
 ///
 /// A `PackedInstruction` in general cannot be safely mutated outside the context of its
 /// `CircuitData`, because the majority of the data is not actually stored here.
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct PackedInstruction {
-    pub op: PackedOperation,
+    op: PackedOperation,
     /// The index under which the interner has stored `qubits`.
-    pub qubits: Interned<[Qubit]>,
+    qubits: Interned<[Qubit]>,
```

Comment:
As discussed we could also keep qubits/clbits public, but I think I agree with your choice here to add methods, such that all members are used in the same fashion.